imagetest/test_suites/guestagent/windows_account_test.go

```diff
@@ -0,0 +1,51 @@
+//go:build cit
+// +build cit
+
+package packagevalidation
+
+import (
+	"fmt"
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+	"strings"
+	"testing"
+)
+
+// Verifies that a powershell command ran with no errors and exited with an exit code of 0.
+// If a username or password was invalid, this should result in a testing error.
+// Returns the standard output in case it needs to be used later.
+func verifyPowershellCmd(t *testing.T, cmd string) string {
+	procStatus, err := utils.RunPowershellCmd(cmd)
+	if err != nil {
+		t.Fatalf("cmd %s failed: stdout %s, stderr %v, error %v", cmd, procStatus.Stdout, procStatus.Stderr, err)
+	}
+
+	stdout := procStatus.Stdout
+	if procStatus.Exitcode != 0 {
+		t.Fatalf("cmd %s failed with exitcode %d, stdout %s and stderr %s", cmd, procStatus.Exitcode, stdout, procStatus.Stderr)
+	}
+	return stdout
+}
+
+func TestCreateNewWindowsUser(t *testing.T) {
+	utils.WindowsOnly(t)
+	username := "windowsuser5"
+	password := "gyug3q445m0!"
+
+	createUserCmd := fmt.Sprintf("net user %s %s /add", username, password)
```

Comment:
Is this what we need to use? This is basically an OS level feature with no guest environment interaction, I was under the impression we need to test the automated windows password reset feature here:

https://cloud.google.com/compute/docs/instances/windows/automate-pw-generation

https://github.com/GoogleCloudPlatform/compute-image-windows/blob/HEAD/examples/windows_auth_go_sample.go

Reply:
Correct - we are aiming to test the guest agent functionality of generating a password and setting it on the guest.